### PR TITLE
3.0: Clean up messages in validators

### PR DESCRIPTION
**Pull request by hanwen-pcluste**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cli/src/pcluster/validators/awsbatch_validators.py [hunk 1]
```diff
@@ -120,17 +117,16 @@ def _validate(self, instance_types, architecture: str):
...
                 compute_architectures = get_supported_architectures_for_instance_type(instance_type)
                 if architecture not in compute_architectures:
                     self._add_failure(
-                        "The specified compute instance type ({0}) supports the architectures {1}, none of which are "
-                        "compatible with the architecture supported by the head node instance type ({2}).".format(
-                            instance_type, compute_architectures, architecture
-                        ),
+                        f"The specified compute instance type ({instance_type}) supports"
+                        f" the architectures {compute_architectures}, none of which is "
+                        f"compatible with the architecture({architecture}) supported by the head node instance type.",
```
Comment (enrico-usai) [inline cli/src/pcluster/validators/awsbatch_validators.py:130]: minor: architecture({architecture}) --> architecture ({architecture})

File: cli/src/pcluster/validators/fsx_validators.py [hunk 2]
```diff
@@ -230,25 +230,25 @@ def _validate(self, auto_import_policy, import_path):
                 if s3_bucket_region is None:
                     s3_bucket_region = "us-east-1"
                 if s3_bucket_region != get_region():
-                    self._add_failure("AutoImport is not supported for cross-region buckets.", FailureLevel.ERROR)
+                    self._add_failure("Auto import is not supported for cross-region buckets.", FailureLevel.ERROR)
```
Comment (enrico-usai) [inline cli/src/pcluster/validators/fsx_validators.py:233]: I'd add "FSx auto import"

File: cli/src/pcluster/validators/fsx_validators.py [hunk 3]
```diff
@@ -230,25 +230,25 @@ def _validate(self, auto_import_policy, import_path):
...
                             bucket, client_error.response.get("Error").get("Message")
                         ),
                         FailureLevel.ERROR,
                     )
                 elif client_error.response.get("Error").get("Code") == "AccessDenied":
                     self._add_failure(
-                        "You do not have access to the S3 bucket '{0}': '{1}'".format(
-                            bucket, client_error.response.get("Error").get("Message")
+                        "You do not have access to the S3 bucket '{0}': '{1}'.".format(
+                            bucket, client_error.response.get("Error").get(".Message")
```
Comment (enrico-usai) [inline cli/src/pcluster/validators/fsx_validators.py:245]: dot added in the wrong place.

**Review by enrico-usai — COMMENTED** [review]
Great job!

File: cli/src/pcluster/validators/awsbatch_validators.py (hunk 1 above)
Comment (hanwen-pcluste) [reply cli/src/pcluster/validators/awsbatch_validators.py]: Done

File: cli/src/pcluster/validators/fsx_validators.py (hunk 3 above)
Comment (hanwen-pcluste) [reply cli/src/pcluster/validators/fsx_validators.py]: Oops, good catch. Done

File: cli/src/pcluster/validators/fsx_validators.py (hunk 2 above)
Comment (hanwen-pcluste) [reply cli/src/pcluster/validators/fsx_validators.py]: Done

**Review by enrico-usai — APPROVED** [review]

**Pull request merged**